packages/@aws-cdk/aws-lambda/lib/log-retention.ts

```diff
@@ -0,0 +1,64 @@
+import iam = require('@aws-cdk/aws-iam');
+import logs = require('@aws-cdk/aws-logs');
+import cdk = require('@aws-cdk/cdk');
+import path = require('path');
+import { Code } from './code';
+import { Runtime } from './runtime';
+import { SingletonFunction } from './singleton-lambda';
+
+/**
```

Comment:
Wouldn’t it make sense to put this construct under aws-logs?

Reply:
This would cause a dependency cycle `@aws-cdk/aws-lambda -> @aws-cdk/aws-log -> @aws-cdk/aws-lambda`, because of `logs.ILogSubscriptionDestination` here:
https://github.com/awslabs/aws-cdk/blob/7001f7723cde611f0d8cd9c182f92120b09579c3/packages/%40aws-cdk/aws-lambda/lib/function-base.ts#L14-L15

Any suggestion on how this could be solved?

Reply:
No, leave it like this. I don't think it's super important (or useful, since you can control a log group you create anyway).